### PR TITLE
More updates for DOCS-1615

### PR DESCRIPTION
**Pull request by jmalin-signalfx**

### CODE REVIEW

File: signalfx-org-metrics/README.md.jinja [hunk 1]
```diff
@@ -61,24 +61,13 @@ the value or limit of MTS or datapoints, whichever caused the detector to stop.
...
-
 ### Metrics that track system limits
 
 Three metrics track limits that Infrastructure Monitoring enforces for all organizations:
 
 * `sf.org.limit.hosts`: Maximum number of hosts that can send data to your organization. If you exceed this limit, Infrastructure Monitoring
   drops datapoints from newly-added hosts but keeps datapoints for existing hosts.
-* `sf.org.limit.containers`: Maximum number of Docker containers that can send data to your organization. If you exceed this limit, Infrastructure Monitoring
+* `sf.org.limit.containers`: Maximum number of containers that can send data to your organization. If you exceed this limit, Infrastructure Monitoring
   drops datapoints from newly-added containers but keeps datapoints for existing containers.
```
Comment (tmorrisonsfx) [inline signalfx-org-metrics/README.md.jinja:71]: Suggested change: "If you exceed this limit, Infrastructure Monitoring drops data points from newly-added containers and keeps data for existing containers."

File: signalfx-org-metrics/metrics.yaml [hunk 2]
```diff
@@ -1364,11 +1364,11 @@ sf.org.limit.customMetricTimeSeries:
   brief: Maximum number of custom MTS you can create in your organization within a 1 hour time window
   description: |
     Maximum number of custom MTS you can create in your organization in a 1 hour time window.
-    If you exceed this limit, Infrastructure Monitoring stops accepting datapoints for new custom MTS,
-    but keeps accepting datapoints for existing custom MTS. To see the overall number of custom metrics
-    you've defined, use the metric `sf.org.numCustomMetrics`.
```
Comment (keitwb) [inline signalfx-org-metrics/metrics.yaml:1369]: "defined" seems right,  why did you change it?

**Review by tmorrisonsfx — APPROVED** [review]
One suggested change

File: signalfx-org-metrics/README.md.jinja (hunk 1 above)
Comment (jmalin-signalfx) [reply signalfx-org-metrics/README.md.jinja:71]: Noted, but at this point I'm going to leave it as is.

File: signalfx-org-metrics/metrics.yaml (hunk 2 above)
Comment (jmalin-signalfx) [reply signalfx-org-metrics/metrics.yaml:1369]: It used to say "define", as in "To see the overall number of custom metrics you've **define**,..."

which is a sad and embarrassing grammar mistake.
